app/src/main/java/org/mozilla/fenix/settings/SettingsFragment.kt

```diff
@@ -39,18 +44,37 @@ class SettingsFragment : PreferenceFragmentCompat() {
         setupPreferences()
     }
 
+    override fun onPreferenceTreeClick(preference: Preference): Boolean {
+        when (preference.key) {
+            resources.getString(pref_key_help) -> run {
```

Comment:
Do we need the run statements here?

Reply:
Doesn't seem like it to me either. Run is useful for changing the meaning of this, but we already have the string in $it and we're not using it either.